.gitignore

```diff
@@ -38,5 +38,5 @@ celeryev.pid
 media/js/libs/ckeditor/source/ckbuilder
 .tox
 .env
-*_devmo.sql*
+*devmo*.sql*
```

Comment:
Should we not just ignore *sql* ? :smile: No point in having SQL in our repo at all.